README.md

```diff
@@ -1,2 +1,66 @@
 # rokt-demo-ios
-Rokt iOS Demo Application
+Rokt Demo application is a sample app built to showcase Rokt SDK functionality.
+
+## Requirements
+
+XCode 13 and above. Project is configured to run on iOS 13.0 and above.
+
+## Project architecture
+
+This project it is implemented based on MVVM pattern with SwiftUI.
+It contains UI, Model, Services, Utils and Resources.
```

Comment:
Explains each

Reply:
A short description of each layer is introduced in the next paragraph.